<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyParticleProcessor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(65, 65, 65);
alternate-background-color: rgb(130, 130, 130);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Datasets</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="treeWidget">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="textElideMode">
            <enum>Qt::ElideRight</enum>
           </property>
           <attribute name="headerVisible">
            <bool>true</bool>
           </attribute>
           <attribute name="headerCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="headerDefaultSectionSize">
            <number>80</number>
           </attribute>
           <attribute name="headerMinimumSectionSize">
            <number>1</number>
           </attribute>
           <attribute name="headerStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Selected</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Step</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Mass</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Charge</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Current</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string># Particles</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string># Steps</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Filename</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Comments</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>7</number>
           </property>
           <item row="0" column="0">
            <widget class="PlotWidget" name="graphicsView_1"/>
           </item>
           <item row="0" column="1">
            <widget class="PlotWidget" name="graphicsView_2"/>
           </item>
           <item row="1" column="0">
            <widget class="PlotWidget" name="graphicsView_3"/>
           </item>
           <item row="1" column="1">
            <widget class="GLViewWidget" name="graphicsView_4"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTextEdit" name="textEdit_2">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImport_New"/>
   <addaction name="actionImport_Add"/>
   <addaction name="actionUnload_Selected"/>
   <addaction name="actionProperties"/>
   <addaction name="actionAnalyze"/>
   <addaction name="actionPlot"/>
   <addaction name="actionExport_For"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>994</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_New"/>
    <addaction name="actionImport_Add"/>
    <addaction name="actionUnload_Selected"/>
    <addaction name="actionExport_For"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionImport_New">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Import New...</string>
   </property>
   <property name="iconText">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New</string>
   </property>
  </action>
  <action name="actionImport_Add">
   <property name="icon">
    <iconset theme="add">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Import Add...</string>
   </property>
   <property name="iconText">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Add</string>
   </property>
  </action>
  <action name="actionExport_For">
   <property name="icon">
    <iconset theme="go-right">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Export For...</string>
   </property>
   <property name="iconText">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionUnload_Selected">
   <property name="icon">
    <iconset theme="remove">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Unload</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="icon">
    <iconset theme="applications-system">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Properties</string>
   </property>
   <property name="toolTip">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionPlot">
   <property name="icon">
    <iconset theme="emblem-photos">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
   <property name="toolTip">
    <string>Plot</string>
   </property>
  </action>
  <action name="actionAnalyze">
   <property name="icon">
    <iconset theme="applications-accessories">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Analyze</string>
   </property>
   <property name="toolTip">
    <string>Analyze</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>GLViewWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph.opengl</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
